qemu/tests/cfg/uefishell.cfg

```diff
@@ -0,0 +1,39 @@
+- uefishell:
+    only q35
+    only ovmf
+    type = uefishell
+    start_vm = no
+    last_error = "lasterror.*=.*0x0"
+    variants:
+        - secureboot:
+            test_scenarios = "enrollkeys"
+            command_enrollkeys = "EnrollDefaultKeys.efi"
+            check_result_enrollkeys = "SecureBootEnable=1"
+        - check_oprom:
+            test_scenarios = "connect oprom"
+            command_connect = "connect -r"
+            command_oprom = "dh -d -v -p SimpleNetwork"
+            check_result_oprom = "Controller Name.*:.*iPXE"
+        - uefi_cmd:
```

Comment:
@YanhuiMa @xiangchunfu Do you think that this case combined too much command and might influence the test result with each other, shall we split them into multiple test cases?

Reply:
@xiangchunfu Currently I have no idea about how to separate these commands, do we have better rules to separate them? 
@vivianQizhu Even if one command fails, we just need to remove it from cfg, I think the cost is acceptable.

Reply:
@vivianQizhu  I don't think it is necessary to spilt them into group or multiple cases. Reasons:
These commands is very simple and the most fundamental command.  and we only check command's execution status.  so they are relatively stable.  once execution has failed.  It will be a product bug.

Reply:
@xiangchunfu Okay, sounds reasonable in case they are stable.